build.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: "jellyfin-plugin-trakt"
 guid: "4fe3201e-d6ae-4f2e-8917-e12bda571281"
-version: "5" # Please increment with each pull request
+version: "6" # Please increment with each pull request
 jellyfin_version: "10.3.6" # The earliest binary-compatible version
```

Comment:
Do we know if this breaks backwards compat?

Reply:
It doesn't.

Reply:
Solid :+1: